frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -280,6 +285,23 @@ function initializeTracing(_annotation: APIAnnotation, tracing: HybridServerTrac
   }
 }
 
+function setInitialTool() {
+  const { useLegacyBindings } = Store.getState().userConfiguration;
+
+  if (!useLegacyBindings) {
+    // The MOVE tool is already the default
+    return;
+  }
+
+  const { tracing } = Store.getState();
+
+  if (tracing.skeleton != null && tracing.volume == null) {
```

Comment:
Wouldn't the current state be more accurately reflected if in legacy mode the skeleton tool would always be the default (except in volume-only)?

Reply:
Yes, good point. Not sure whether there was a reason against that. We probably missed that variation or I mixed something up. Do you agree, @fm3? I implemented the change in 3dafe17f2fcdc5e8c76fc0889fa712681a012be2.

Reply:
I don’t remember exactly what we discussed about this previously, but yes, seems fair to me right now.